recipes/osx-sysroot/conda_build_config.yaml

```diff
@@ -0,0 +1,51 @@
+_MACOSX_DEPLOYMENT_TARGET_:
+  - "10.9"
+  - "10.13"
+  - "11.0"
+  - "11.0"
+  - "12.3"
+
+cross_target_platform:
+  - osx-64
+  - osx-64
+  - osx-64
+  - osx-arm64
+  - osx-arm64
+
+_MACOSX_SDK_VERSION_:
+  - "1"
+  - "10.9"
+  - "10.13"
+  - "11.0"
+  - "12.3"
```

Comment:
I find the mix between version schemes here quite jarring. I also think we should have osx-64 for all versions where we have osx-arm64 (well, as long as apple supports that).

Reply:
Sorry yeah. The list of versions here is not complete on purpose. It made the builds faster to test. 

Version 1 is the system SDK. I didn't know how else to make that one work and version 0 is not allowed. 

We have planned to add support for the system sysroot on linux but haven't gotten around to it.